.eleventy.js

```diff
@@ -65,13 +66,16 @@ module.exports = function(eleventyConfig) {
                     const code = token.content.trim();
                     if (code && code.toLowerCase().startsWith("title:")) {
                         const title = code.substring(6, code.indexOf("\n"));
-                        const titleDiv = title ? `<div class="admonition-title">${title}</div>` : '';
```

Comment:
I haven't used the plugin or this style of admonition before so I don't know if I did this right

Reply:
This should be fine. Admonition-title was the internal class name used when callouts wasn't a thing and admonition was the way to go. I simply hadn't bothered to change it. "callout-title" is more correct as it is now.